src/spikeinterface/preprocessing/motion.py

```diff
@@ -280,7 +293,8 @@ def correct_motion(
             pipeline_nodes,
             job_kwargs,
             job_name="detect and localize",
-            gather_mode=gather_mode,
+            gather_mode="memory",
```

Comment:
Can you put back this gather_mode before in a variable  like before ?

Reply:
ok

Reply:
done